client/src/components/payment/payment-complete-modal.vue

```diff
@@ -140,17 +140,19 @@ export default class PaymentCompleteModal extends Mixins(NameRequestMixin, Payme
 
   async completePayment (nrId: number, paymentId: number, action: string) {
     const result: NameRequestPayment = await newRequestModule.completePayment({ nrId, paymentId, action })
-    const paymentSuccess = result.paymentSuccess
+    const paymentSuccess = result?.paymentSuccess
 
     if (paymentSuccess) {
       paymentModule.toggleReceiptModal(true)
-    } else if (!paymentSuccess && result.paymentErrors) {
+    } else if (!paymentSuccess && result?.paymentErrors) {
       // Setting the errors to state will update any subscribing components, like the main ErrorModal
       await errorModule.setAppErrors(result.paymentErrors)
       if (action && action === PaymentAction.COMPLETE) {
         // Cancel the NR using the rollback endpoint if we were processing a NEW NR
-        await newRequestModule.rollbackNameRequest({ nrId, action: rollbackActions.CANCEL })
+        await newRequestModule.rollbackNameRequest({ nrId, action: RollbackActions.CANCEL })
       }
+    } else {
+      console.log('completePayment(), unhandled scenario, result =', result) // eslint-disable-line no-console
```

Comment:
Is this something you'd want Sentry to see?

Reply:
You've changed my mind. Yes, I will update.

Reply:
Fixed.